Dev/v1.2

alexjhawk opened the pull request.

TomKimsey reviewed (changes requested):
It would be beneficial to have more class documentation for DataPointIntegerEnum. Maybe even giving an example or reason for why to use it. Also the class name seems a bit confusing. Isnt the meaningful value datatype really a string and not an int?

ex
0 = "Off"
1= "On, Not Running"
2 = "On, Running"
3 = "On, Not Running, Faulted"

To me string better defines those datatypes than integer. Of maybe the class should be something more like DataPointIntegerEncodedString

Edit: Encoded is probably not the right word to use 
Maybe DataPointIntegerMappedString

alexjhawk commented:
> It would be beneficial to have more class documentation for DataPointIntegerEnum. Maybe even giving an example or reason for why to use it. Also the class name seems a bit confusing. Isnt the meaningful value datatype really a string and not an int?
> 
> ex
> 0 = "Off"
> 1= "On, Not Running"
> 2 = "On, Running"
> 3 = "On, Not Running, Faulted"
> 
> To me string better defines those datatypes than integer. Of maybe the class should be something more like DataPointIntegerEncodedString
> 
> Edit: Encoded is probably not the right word to use
> Maybe DataPointIntegerMappedString

- Renamed new data point class to your suggested name "DataPointIntegerMappedString" because I feel that fits well. 
- Added example in class documentation

TomKimsey commented on CHANGELOG.md:
```diff
@@ -0,0 +1,25 @@
+
+## Version 1.2
+### Features
+- Added integer enumeration data point type
```
needs to get updated for new class name

alexjhawk replied:
updated
